ISHLogDNAService/ISHLogDNAService.m

```diff
@@ -72,8 +72,8 @@ + (instancetype)messageWithLine:(NSString *)line level:(ISHLogDNALevel)level met
         [errorDict setObject:error.domain forKey:ISHLogDNAServiceKeyErrorDomain];
     }
 
-    if (error.description.length) {
-        [errorDict setObject:error.description forKey:ISHLogDNAServiceKeyErrorDescription];
+    if (error.localizedFailureReason.length) {
+        [errorDict setObject:error.localizedFailureReason forKey:ISHLogDNAServiceKeyErrorDescription];
```

Comment:
We might want to log the unlocalized failure reason to avoid having the same log in different languages.